napari/components/_viewer_constants.py

```diff
@@ -1,5 +1,9 @@
 from enum import Enum
 
+from napari.utils.translations import trans
+
+DEFAULT_STATUS_TEXT = trans._('Ready')
```

Comment:
@goanpeca : is it OK to define a constant translated string? Or should we expect to respond to a locale change during of napari?

Based on the `napari.utils.translations` module, it looks like it is because `trans._` is initialized at import time and is tied to the domain/locale at that time.